tool_quality_checks/dc_input_create.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+# TODO begin- en eindputcode is niet beschikbaar voor putten en kunstwerken

```

Comment:
Deze mag denk ik uit de checks rollen en hoeft niet perse in deze view te staan?
Wat denk jij?